src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1497,7 +1497,7 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>'{0}' in explicit interface declaration is not an interface</value>
   </data>
   <data name="ERR_InterfaceMemberNotFound" xml:space="preserve">
-    <value>'{0}' in explicit interface declaration is not a member of interface</value>
+    <value>'{0}' in explicit interface declaration is not found among members of the interface that can be implemented</value>
   </data>
```

Comment:
> I agree, given that you now use the same diagnostic for other situations

With regard to this error message, I see very little difference in expanding the situation from just static members to non-virtual members in general. The original message was inaccurate for scenarios that could arise before this feature. I didn't make it vague, I made it more accurate, it simply was incorrect for static members and they were allowed.